### PR TITLE
Profile uploads

### PR DESCRIPTION
**Pull request by StoneT2000**
Allows users to edit their profile picture

Requires the backend merge first

### CODE REVIEW

File: src/components/ProfileCard/index.jsx [hunk 1]
```diff
@@ -3,12 +3,16 @@ import PropTypes from 'prop-types';
 import { Avatar, Card, Progress } from 'antd';
 
 import './style.less';
+import { getDefaultProfile } from '../../utils';
 
 const ProfileCard = props => {
+  React.useEffect(() => {
+    console.log(props.profilePicture);
```
Comment (leestanley) [inline src/components/ProfileCard/index.jsx:10]: prod console logs

File: src/components/ProfileUpdate/index.jsx [hunk 2]
```diff
@@ -16,6 +19,50 @@ const years = getYears();
...
+    URL.revokeObjectURL(bg);
+    if (fileList.length) {
+      let lastFile = fileList[fileList.length - 1];
+      if (lastFile) {
+        setFileList([lastFile]);
+      }
+      let newBg = URL.createObjectURL(lastFile.originFileObj);
+      console.log(newBg);
+      setBG(newBg);
+      console.log(lastFile);
```
Comment (leestanley) [inline src/components/ProfileUpdate/index.jsx:42]: prod console logs

Comment (StoneT2000) [reply src/components/ProfileUpdate/index.jsx:42]: oh woops

Comment (StoneT2000) [reply src/components/ProfileUpdate/index.jsx:42]: good catches

**Pull request merged**